fastlane/lib/fastlane/actions/app_store_build_number.rb

```diff
@@ -22,7 +24,9 @@ def self.run(params)
           unless version_number
             # Automatically fetch the latest version in testflight
             begin
-              testflight_version = app.build_trains.keys.last
+              # this is working in irb but is giving me an exception when called here
+              # exception is: ``platform' is a required parameter`
```

Comment:
Is the exception still happening when called from outside irb, or is this comment about why you changed the code?

Reply:
Comment was stale from before I adjusted `all_build_train_numbers`. Thanks for flagging. Amended.